tests/generator/test_uuid.py

```diff
@@ -0,0 +1,22 @@
+import uuid
+
+from unittest import TestCase
+from unittest.mock import MagicMock, patch
+
+from flask_seeder.generator import UUID
+
+
+class TestUUIDGenerator(TestCase):
+    def setUp(self):
+        self.generator = UUID()
+
+    @patch("uuid.uuid4", return_value=0)
+    def test_generate_mock_uuid(self, m_uuid):
+        result = self.generator.generate()
+
+        assert result == 0
+
+    def test_generate_uuid(self):
+        result = self.generator.generate()
+
+        uuid.UUID(str(result), version=4)
```

Comment:
Could you make an assertion about the expected result here? `UUID()` will raise an exception if it's not a valid uuid so the test would fail, but making an explicit assertion will help others reading the test understand the intention behind it (it may not be obvious that `UUID()` raises an exception simply by reading the test). It also helps in the test log output as it will print a more sensible error message.

Something like `self.assertEqual(4, uuid.UUID(str(result)).version)` or similar.

Reply:
Makes sense! Done and done :)